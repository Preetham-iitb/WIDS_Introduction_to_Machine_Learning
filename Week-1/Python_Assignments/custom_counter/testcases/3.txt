This is a random testcase
a1!s2@d3#j4$f5%h6^w7&e8*r9(o0)q-w_p=r+o1
{i}u2[q]z3|x/c4>m5<a,6s.l?d7*j~f;8q"w'o:9i`e~r!u0@
y^t*z1&x(2n)m#3a$s%4l^k*5d(j)f_p-6o=e[7r]u{8t}y|9x/0z
,c<1v2m>a3<l>s4>k.d5!j@6f#h$q7^p*o(8w)
e-9i=r+u[0t]y{1z}|n~2m/c>3a,s.4l?k"5d'j:f;6qw`7`
