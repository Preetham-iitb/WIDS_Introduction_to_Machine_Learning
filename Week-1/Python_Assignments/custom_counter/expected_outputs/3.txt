a : 8
c : 4
d : 6
e : 6
f : 5
h : 3
i : 5
j : 5
k : 3
l : 4
m : 5
n : 3
o : 6
p : 3
q : 5
r : 6
s : 9
t : 5
u : 4
v : 1
w : 5
x : 3
y : 3
z : 4
T : 1
0 : 4
1 : 5
2 : 5
3 : 5
4 : 5
5 : 5
6 : 5
7 : 5
8 : 4
9 : 4
  : 4
` : 3
~ : 3
! : 3
@ : 3
# : 3
$ : 3
% : 2
^ : 4
& : 2
* : 5
( : 4
) : 4
_ : 2
+ : 2
- : 3
= : 3
[ : 3
] : 3
{ : 3
} : 3
| : 3
; : 2
' : 2
" : 2
. : 3
> : 5
, : 3
< : 3
/ : 3
? : 2
